Assets/Scripts/PlayerMovement.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.InputSystem;
+using UnityEngine.PlayerLoop;
+
+public class PlayerMovement : MonoBehaviour
+{
+    public PlayerTPLocationDefinition PlayerTP;
+    public static float SpeedMultiplier = 10;
+    
+    public bool isInPast = true;

```

Comment:
It looks like you are using CamelCase for public variables, then can we make this `IsInPast`?

Reply:
Any feedback on this comment?